django_api_forms/fields.py

```diff
@@ -301,3 +301,26 @@ def to_python(self, value) -> typing.Optional[File]:
         f.seek(0)  # Return to start of the file
 
         return f
+
+
+class RRuleField(Field):
+    default_error_messages = {
+        'not_rrule': _('This field needs to be a rrule object!'),
+        'invalid_rrule': _('This given RRule String is not in a valid RRule syntax.'),
+    }
+
+    def __init__(self, **kwargs) -> None:
+        super().__init__(**kwargs)
+
+    def to_python(self, value: str):
+        # Dateutil is required for RRuleField
+        from dateutil.rrule import rrulestr
+
+        try:
+            result = rrulestr(value)
+        except Exception:
+            raise ValidationError(
+                self.error_messages['invalid_rrule']
```

Comment:
You also should specify the `code` attribute (`invalid_rrule` for example)

Reply:
@paimvictor maybe you forgot to push this change

Reply:
My bad, just sent it again